lib/hexpm/web/views/error_view.ex

```diff
@@ -32,7 +32,7 @@ defmodule Hexpm.Web.ErrorView do
   end
 
   defp message("400"), do: "Bad request"
-  defp message("404"), do: "Page not found"
+  defp message("404"), do: "The page you’re looking for doesn’t exist"
```

Comment:
The same error message is used for the API so I think we need to keep the current message or match on the "html" format.